tests/integration/test_k8s.py

```diff
@@ -19,215 +19,214 @@
 """
 
 import urllib3
+import pytest
 
 from conu import DockerBackend
-from conu.backend.k8s.backend import K8sBackend
-from conu.backend.k8s.backend import K8sCleanupPolicy
+from conu.backend.k8s.backend import K8sBackend, K8sCleanupPolicy
 from conu.backend.k8s.pod import PodPhase
 from conu.backend.k8s.service import Service
 from conu.backend.k8s.deployment import Deployment
 from conu.backend.k8s.client import get_core_api
-from conu.utils import get_oc_api_token
-
+from conu.utils import get_oc_api_token, oc_command_exists, is_oc_cluster_running
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
-def test_pod():
-    api_key = get_oc_api_token()
-    with K8sBackend(api_key=api_key) as k8s_backend:
+@pytest.mark.skipif(not oc_command_exists(), reason="OpenShift is not installed!")
+@pytest.mark.skipif(not is_oc_cluster_running(), reason="OpenShift cluster is not running!")
```

Comment:
When I was trying to use two `@pytest.mark.skipif`s some time ago it didn't seem to work as expected and I failed back to using only one. So please make sure this works as you expect, thanks.